src/Lean/Meta/Tactic/AC/Main.lean

```diff
@@ -101,11 +101,11 @@ def buildNormProof (preContext : PreContext) (l r : Expr) : MetaM (Lean.Expr ×
 where
   mkContext (α : Expr) (u : Level) (vars : Array Expr) : MetaM (Array Bool × Expr) := do
     let arbitrary := vars[0]!
-    let zero := mkLevelZeroEx ()
-    let plift := mkApp (mkConst ``PLift [zero])
-    let pliftUp := mkApp2 (mkConst ``PLift.up [zero])
-    let noneE tp   := mkApp  (mkConst ``Option.none [zero]) (plift tp)
-    let someE tp v := mkApp2 (mkConst ``Option.some [zero]) (plift tp) (pliftUp tp v)
+    let lvlZero := mkLevelZeroEx ()
+    let plift := mkApp (mkConst ``PLift [lvlZero])
+    let pliftUp := mkApp2 (mkConst ``PLift.up [lvlZero])
+    let noneE tp   := mkApp  (mkConst ``Option.none [lvlZero]) (plift tp)
+    let someE tp v := mkApp2 (mkConst ``Option.some [lvlZero]) (plift tp) (pliftUp tp v)
```

Comment:
```suggestion
    let plift := mkApp (mkConst ``PLift [.zero])
    let pliftUp := mkApp2 (mkConst ``PLift.up [.zero])
    let noneE tp   := mkApp  (mkConst ``Option.none [.zero]) (plift tp)
    let someE tp v := mkApp2 (mkConst ``Option.some [.zero]) (plift tp) (pliftUp tp v)
```

I think this can be simplified further to just use the constructor directly. `mkLevelZeroEx` is iexport of the constructor for use in C code. There is no reason for it have been used it here.  Thus, this serves to further demonstrate the utility of this lint. 😄

Reply:
I saw that while reading the code - figured Chesterton's Fence meant I shouldn't tweak it. Let's see if the tests still pass with this suggestion committed - it would be surprising if this broke something, after all!